.travis.yml

```diff
@@ -40,6 +40,14 @@ stages:
 
 jobs:
   include:
+  # fedora (released fedora with openmpi)
```

Comment:
fedora / bionic 
please add a comment explaining why we mix the two :)

Reply:
no idea why ! not me !

Reply:
Not sure what the question is?
Why fedora? Why openmpi? Why released?

I added an openmpi and an mpich case, to test both mpi versions, and also an released and still-in-development version of fedora, also to increase coverage. Why I mixed the on with the other, no real reason.

Reply:
Not sure what the question was either ! Good for me.